keras_nlp/tokenizers/sentence_piece_tokenizer_trainer.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Trainer for SentencePiece Tokenizer."""
+import io
+
+import tensorflow as tf
+
+try:
+    import sentencepiece as spm
+except ImportError:
```

Comment:
Let's print a warning, an example could be "sentencepiece pip package is not found, please do `pip install sentencepiece` and rerun the script".